src/Tools/MicroBuild/cibuild.ps1

```diff
@@ -22,7 +22,11 @@ try {
     Exec { & $nuget locals all -clear }
 
     $msbuild = Ensure-MSBuild
-    & $msbuild /nodereuse:false /p:Configuration=Release /p:SkipTest=true Build.proj 
+
+    # The /nowarn exception can be removed once we fix the following:
+    #  - 3277: https://github.com/dotnet/roslyn/issues/17325
+    #  - 4011: https://github.com/dotnet/roslyn/issues/18151
+    & $msbuild /nodereuse:false /p:Configuration=Release /p:SkipTest=true Build.proj /warnaserror /nowarn:3277,4011
```

Comment:
@tannergooding is correct.  Microbuild enters us directly at Build.proj.  In Jenkins cibuild.ps1 drives our entry into Build.proj.